src/hammer-vlsi/defaults.yml

```diff
@@ -118,6 +118,7 @@ vlsi.inputs:
   #   name (str) - The name of your supply's net
   #   pin (Optional[str]) - The cell pin that this net should drive, no pins are connected when omitted
   #   tie (Optional[str]) - Another supply this supply is tied to. Nets with this set will  not end up in your final layout.
+  #   weight (int) - The weight of this supply when building power straps (larger number = more straps). Not used for grounds. Mandatory, but defaults to 1 if omitted.
```

Comment:
Yeah, I wasn't sure what to do here. The underlying `NamedTuple` is of type `int` because there is a default. The goal here was to avoid breaking existing yamls. I am open to suggestion on what to do.

Reply:
I think the comment clarifies that there is a default value so maybe it's fine?

Reply:
Fixed.